### ISSUE TITLE
Installation Problems

### ISSUE THREAD

**Issue by alex-seitz** [imgag/megSAP#160]
Hi,
I set up a new instance of megSAP a few days ago and ran into some problems/warnings:
- When installing the RNA-tools, Conda also needs to be installed but it is not present in the pre-requisites
- `download_tools.sh`
    - I'm on Ubuntu 20.04. There spliceai seems to need Python Version 3.7.0 `SyntaxError: future feature annotations is not defined` This apparently works only in Python 3.7.0, the downloaded version is 3.6.9
    - When I tried it with 3.7.0, I got another error when installing cyvcf2 that I need at least Python 3.8.
    - I then tried it with the newest Version 3.9.9 and needed to install the following packages
        - `sudo apt install pkg-config libhdf5-dev`
        - `pip install tensorflow`in the `splice_env` generated by the install script
    - Now it seems to work
-  `download_dbs.sh`
    - ClinVar: the link is incorrect (the directory still states `2021` instead of `2022`)
    - CADD: The wrong type of "-" is present in both of the VcfCheck commands
    - CADD: You get a lot of errors "Reference base(s) not correct. Is 'C', should be 'N'! - in line XXX" I guess this comes from the adapted Genome where the false duplications are marked?!
    - REVEL: The website `https://rothsj06.u.hpc.mssm.edu/` no longer seems to exist
- When running the analyses, I get the warning `Annotation of structural variants is outdated` when openign the analyses in GSvar
    - I also get the `WARNING: 'Cannot annotate NGSD counts! At least one required file in ...data//dbs/NGSD/ is missing!`.
    - This seems to be, because the `$ngsd_sv_files` and the `sv_breakpoint_density.igv` files are missing
    - How do I get/generate them?
 
I also get the following warning: `PHP Warning:  "continue" targeting switch is equivalent to "break". Did you mean to use "continue 2"? in /shared_software/megSAP_GRCh38_20221026/src/NGS/db_import_qc.php on line 92` Is that relevant?

Thank you for your time

Best,
Alex

**Comment by marc-sturm** [comment]
Revel is fixed: https://github.com/imgag/megSAP/commit/cc96c95a6ceb6b130c3f24ec51b96a19a3f44e6e

**Comment by marc-sturm** [comment]
ClinVar path is fixed: https://github.com/imgag/megSAP/commit/44b688551ff782e940eaf07aad1410f775216000

This also fixes the CADD errors and restricts the VCF check to the first 1000 lines. Then warnings caused by masking false duplications should no longer occur.

**Comment by marc-sturm** [comment]
The continue warning is already fixed: https://github.com/imgag/megSAP/commit/2d1638babc4c1ac219d673de8cf34cc424769753

**Comment by marc-sturm** [comment]
I added documentation for the export of SV data from NGSD: ce4d67369f7f100af5e391f409a9e132fc9b41be

**Comment by marc-sturm** [comment]
The SpliceAI issue with Python is very strange.
We install a fixed version of python and spliceAI, so they always should work together:
https://github.com/imgag/megSAP/blob/master/data/download_tools.sh

The Python 3.7 from the package manager should never be called.
Can you provide more information? E.g. a log file?

**Comment by alex-seitz** [comment]
Thank you for the quick and thorough explanations.
For the python issue, I will call the installation process on our second client and will post you the results.
I did just call that script, but got the errors.
Like I said, I will call it on the second client and will keep you posted

**Comment by alex-seitz** [comment]
I created a minimal script for the installation for python based on the `download_tools.sh` to reproduce the error. I attached it with the name `test_install.sh`
What I realized that I had to change the prerequisites for megSAP to change all the python2.7 to python3 modules (e.g. python-matplotlib to python3-matplotlib).
[python_install.zip](https://github.com/imgag/megSAP/files/10121715/python_install.zip)

I'm guessing that in one of those prerequisites lies the error.
I also attached the complete installation log from the test install.
I had to put both files in a zip file, as I'm not allowed to upload `.sh` files
